src/sentry/static/sentry/app/utils/discover/arrayValue.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+import space from 'app/styles/space';
+import {t} from 'app/locale';
+
+type Props = {
+  value: string[];
+};
+type State = {
+  expanded: boolean;
+};
+
+class ArrayValue extends React.Component<Props, State> {
+  state: State = {
+    expanded: false,
+  };
+  handleToggle = () => {
+    this.setState(prevState => ({
+      expanded: !prevState.expanded,
+    }));
+  };
+
+  render() {
+    const {expanded} = this.state;
+    const {value} = this.props;
+    return (
+      <ArrayContainer>
+        {expanded &&
+          value
+            .slice(0, value.length - 1)
+            .map(item => <ArrayItem key={item}>{item}</ArrayItem>)}
```

Comment:
Will item here always be unique to be used as a key? I believe that it's possible for two items to be equal for some array fields.

Reply:
Good point, I'll use the `item,index` instead.